tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -313,9 +313,9 @@ dualtor/test_standby_tor_upstream_mux_toggle.py:
 #######################################
 dut_console:
   skip:
-    reason: "Not supported vs testbed."
-    conditions:
-      - "asic_type in ['vs']"
+    reason: "Not supported vs testbed. / Public console account is not available."
+#    conditions:
```

Comment:
@yutongzhang-microsoft There are two # at the head of these 2 lines, is it on purpose? If DUT is vs, we should skip it I think.

Reply:
It is on purpose, it will skip on all testbeds.